py3-jupyterhub-ldapauthenticator.yaml

```diff
@@ -31,22 +33,47 @@ pipeline:
       repository: https://github.com/jupyterhub/ldapauthenticator
       tag: ${{package.version}}
 
-  - name: Python Build
-    uses: python/build-wheel
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: python${{range.key}} version of ${{vars.pypi-package}}
+    dependencies:
+      provider-priority: ${{range.value}}
+      provides:
+        - py3-${{vars.pypi-package}}
+      runtime:
+        - py${{range.key}}-jupyterhub
+        - py${{range.key}}-ldap3
+        - py${{range.key}}-tornado
```

Comment:
```bash
$ uv pip show jupyterhub-ldapauthenticator
Name: jupyterhub-ldapauthenticator
Version: 2.0.2
Location: /Users/anurag.kumar/projects/uv-project/.venv/lib/python3.11/site-packages
Requires: jupyterhub, ldap3, traitlets
Required-by:
```
looks like tornado is not required here or was it required by any older python version.

Reply:
Good catch! Let me remove it.